src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+
```

Comment:
For the future, remove the first empty line in all classes/interfaces/methods where it is present
```suggestion
```